README.md

```diff
@@ -37,20 +37,20 @@ Or add it as a script to your `package.json`
 
 This is a combination of the commands suggested in the React Native documentation plus others.
 
-| State Type           | Command                       | In `clean-project-auto`? | Optional?  | Default? | Option Flag            |
-| -------------------- | ----------------------------- | ------------------------ | ---------- | -------- | ---------------------- |
-| React-native cache   | `rm -rf $TMPDIR/react-*`      | Yes                      | No         | true     |                        |
-| Metro bundler cache  | `rm -rf $TMPDIR/metro-*`      | Yes                      | No         | true     |                        |
-| Watchman cache       | `watchman watch-del-all`      | Yes                      | No         | true     |                        |
-| NPM modules          | `rm -rf node_modules`         | Yes                      | Yes        | true     | --keep-node_modules    |
-| Yarn cache           | `yarn cache clean`            | Yes                      | Yes        | true     | --keep-node-modules    |
-| Yarn packages        | `yarn install`                | No                       | Yes        | true     | --keep-node-modules    |
-| NPM cache            | `npm cache verify`            | Yes                      | Yes        | true     | --keep-node-modules    |
-| iOS build folder     | `rm -rf ios/build`            | Yes                      | Yes        | false    | --remove-iOS-build     |
-| iOS pods folder      | `rm -rf ios/pods`             | Yes                      | Yes        | false    | --remove-iOS-pods      |
-| Android build folder | `rm -rf android/build`        | Yes                      | Yes        | false    | --remove-android-build |
-| Brew package         | `brew update && brew upgrade` | No                       | Yes        | true     | --keep-brew            |
-| Pod packages         | `pod update`                  | No                       | Yes        | true     | --keep-pods            |
+| State Type           | Command                       | In `clean-project-auto`? | Optional? | Default? | Option Flag                                 |
+| -------------------- | ----------------------------- | ------------------------ | --------- | -------- | ------------------------------------------- |
+| React-native cache   | `rm -rf $TMPDIR/react-*`      | Yes                      | No        | true     |                                             |
+| Metro bundler cache  | `rm -rf $TMPDIR/metro-*`      | Yes                      | No        | true     |                                             |
+| Watchman cache       | `watchman watch-del-all`      | Yes                      | No        | true     |                                             |
+| NPM modules          | `rm -rf node_modules`         | Yes                      | Yes       | true     | --keep-node_modules --remove-node_modules   |
```

Comment:
Having these two side-by-side could perhaps draw some confusion. Is there an alternative to this?